tests/backend/user/test_apiv2_human_find.py

```diff
@@ -311,3 +323,14 @@ async def test_bad_args(access_testbed, organization_factory, local_device_facto
     for bad in [{"page": 0}, {"per_page": 0}, {"per_page": 101}]:
         rep = await human_find(sock, **bad)
         assert rep["status"] == "bad_message"
+
+
+@pytest.mark.trio
+async def test_human_handle_to_none(access_testbed, organization_factory, local_device_factory):
```

Comment:
Better call it "non-human user" instead of "human handle to none" (that what is used in the rest of the codebase)

Reply:
so the test should be called `test_find_with_query_ignore_non_human`

Reply:
done